### DISCUSSION TITLE
Can't enable dependabot for version updates

### DISCUSSION THREAD

**Discussion by max06** [community/community#60386] [Code Security] [answered]
### Select Topic Area

Bug

### Body

Hello there,
I'm currently trying to setup dependabot for version updates in a forked ruby project. The option for that is shown in the repo settings, is clickable and even confirms settings have been saved. Yet, the button stays on "enable" and nothing happens.
The configuration file is [here](https://github.com/max06/just-the-docs-template/edit/main/.github/dependabot.yml). 
All the other options are enabled.
![image](https://github.com/community/community/assets/7556827/25e963b2-4708-4121-8e28-cc30bf3d0472)
Did I do something wrong? Did I miss something?

**Comment by carogalvin** [comment]
@max06 does Dependabot have any logs in the Dependency Graph section? https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#checking-the-status-of-version-updates

**Reply by max06** [reply]
@carogalvin thx for your response!

![image](https://github.com/community/community/assets/7556827/7cbdc1f9-1a4c-4aa8-a318-218e43085ac9)

Clicking the button brings me to my existing dependabot configuration file...

Edit: I invited you to the repo so you can debug/check for yourself.

**Reply by carogalvin** [reply, marked as answer]
Oh, I think I see the problem. You have `schedule: daily` when what you want is:
```
    schedule:
      interval: "daily"
```
Try that and see if it starts working.

**Reply by max06** [reply]
Oh no 🙈 

You're right. And there seems to be a second issue: `The property '#/updates/0/allow' of type object did not match the following type: array`. After fixing both the dependency graph suddenly shows content.

If I'm not mistaken, you can't enable the dependabot version updates with an invalid configuration file - and you don't get to see configuration errors until dependabot version updates are enabled. Kind of a loophole, and I ran right into it...

Shame on me for not checking the docs more thoroughly - and thank you for your help 🙇🏼

**Reply by carogalvin** [reply]
No problem, I'm glad to hear that it's working for you now!

**Comment by Yash-Dobariya** [comment]
schedule:
      interval: "daily"

Try
